.teamcityTest/Gradle_Check_Tests/CIConfigIntegrationTests.kt

```diff
@@ -79,11 +79,11 @@ class CIConfigIntegrationTests {
     fun functionalTestsCanDependOnSpecificBuilds() {
```

Comment:
❌ This test does not test anymore what it should test, since every build has a dependency on `SanityCheck`. Either you add a specific build and enable the test again or you delete the test. WDYT?

Reply:
Test removed